src/Select/index.js

```diff
@@ -184,6 +172,22 @@ class Select extends Component {
         return this.refs.menu;
     }
 
+    getButtonText() {
+        const value = this.props.value;
+        const text = this.getItems().reduce((res, item) => {
+            if (value.indexOf(item.props.value) !== -1) {
+                if (value.length === 1) {
+                    res.push(Component.textValue(item));
+                } else {
+                    res.push(item.props.checkedText || Component.textValue(item));
+                }
+            }
+            return res;
+        }, []);
+
+        return text.join(', ') || this.props.placeholder || '—';
```

Comment:
If we want this method to be overridable, I think there is nothing particular wrong for it to be here.